test/tst_sampling.jl

```diff
@@ -1,13 +1,14 @@
 using StatsBase
 using Base.Test
 using MLDataUtils
+using DataStructures
```

Comment:
I removed `DataStructures` from `REQUIRE`, so we would need to add it to `test/REQUIRE` if used in the tests